cypress/component/annot-continuous-values.cy.js

```diff
@@ -0,0 +1,83 @@
+import annotContinuousValues from "~/components/annot-continuous-values";
+
+
+const store = {
+    getters: {
+        getPreviewValues: () => (activeCategory) => {
+            return {
+                "column1": ["1Y", "11Y"],
+                "column2": ["2,1", "22,1"]
+            };
+        },
+        getHarmonizedPreview: () => (column, missingValue) => null,
+        getTransformHeuristics: () => (activeCategory) => {
+            return ["float", "bounded", "euro", "range", "int", "string", "isoyear"];
+        },
+        getActiveHeuristic: () => (activeCategory) => null
+    }
+};
+
+
+const props = {
+    activeCategory: "category1"
+};
+
+describe("continuous-values-component", () => {
+        it("correctly displays preview values provided by the store", () => {
+                cy.mount(annotContinuousValues, {
+                    propsData: props,
+                    computed: store.getters,
+                    plugins: ["vue-select"]
+                });
+                cy.get('.table').then(table => {
```

Comment:
We should not be relying on class names from the app since they are subject to change.

The recommended means of getting an element on the page is to define a separate attribute for cypress (e.g. `data-cy`).

Then when getting the element in the test code use:

`cy.get("[data-cy='attributeValue']")`

Reply:
Thanks, good catch. This was me being lazy and not catching it! I'll give the table a cypress tag!